### PR TITLE
Add `getThumbnail` APIs to `Player` and `Source`

### PR DESCRIPTION
**Pull request by rolandkakonyi**
## Description

This PR extends the `Player` and `Source` APIs to allow fetching thumbnail information.

## Changes

Introduced:
- `Player.getThumbnail()`: to fetch thumbnail information for the current source
- `Source.getThumbnail()`: to fetch thumbnail information for the given source

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Support to get thumbnail images via `Player.getThumbnail` and `Source.getThumbnail`
```
Comment (zigavehovec) [inline CHANGELOG.md:7]: I would prefer the changelog style that you used in #224 

```suggestion
- `Player.getThumbnail` and `Source.getThumbnail` APIs to get thumbnail images
```

File: android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt [hunk 2]
```diff
@@ -428,6 +428,18 @@ class PlayerModule(private val context: ReactApplicationContext) : ReactContextB
...
 
+    /**
+     * Returns the thumbnail image for the active `Source` at a certain time.
+     * @param nativeId Target player id.
+     * @param time Playback time for the thumbnail.
+     */
+    @ReactMethod
+    fun getThumbnail(nativeId: NativeId, time: Double, promise: Promise) {
+        uiManager()?.addUIBlock {
+            promise.resolve(JsonConverter.fromThumbnail(players[nativeId]?.source?.getThumbnail(time)))
```
Comment (zigavehovec) [inline android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt:439]: `Player.getThumbnail` has been deprecated in favour of `Source.getThumbnail` on Android. From my perspective I would be in favour of only wrapping `Source.getThumbnail` in this PR, but I'm not sure what the state is on iOS, just wanted to point this out.

**Review by zigavehovec — COMMENTED** [review]
Nice work 👍

File: CHANGELOG.md (hunk 1 above)
Comment (rolandkakonyi) [reply CHANGELOG.md:7]: Good point!
Taking the suggested change!

File: android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt (hunk 2 above)
Comment (rolandkakonyi) [reply android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt:439]: @zigavehovec I know about the deprecation on Android but I went for providing both as our `Source` API is not that rich in the React Native SDK yet. Also we don't support playlist, so only one `Source` is available at a time.

**Review by zigavehovec — APPROVED** [review]

**Pull request merged**